code/src/App.js

```diff
@@ -1,12 +1,35 @@
 import React from 'react'
 import data from './data.json'
+import { Album } from './Album'
+// import { Artist } from './Artist'
+
+import './style.css'
+import './index.css'
+import './mediaqueries.css'
+import './icons.css'
+import './Album.css'
+
+const music = data.albums.items
+
 
 console.log(data)
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+
+    <div className="wrapper">
+      <h1>New releases - albums &amp; singles</h1>
+      <div className="musicContainer">
+        {music.map((album) => {
+          return <Album
+            key={album.name + album.artists[0].name}
+            url={album.external_urls.spotify}
+            src={album.images[0].url}
+            songName={album.name}
+            artistUrl={album.artists[0].external_urls.spotify}
+            songArtist={album.artists[0].name} />
```

Comment:
Regarding the artists.
Since some of the albums have multiple artists you should make sure to show all of them.
Pass along the whole artists-array as a prop, so instead of songArtist and artistUrl go for artists={album.artists}

Reply:
Thanks a lot for the feedback!